docs/faq.rst

```diff
@@ -170,9 +170,9 @@ User may observe the error: "Failed to get a new subscription. Subscription Refr
 Other
 -----
 
-**Q. How can I change the port of a BIG-IP device which is already added in the F5 ACI ServiceCenter?**
+**Q. How can I change the management port of a BIG-IP device which is already added in the F5 ACI ServiceCenter?**
 
-Click the delete (X) icon besides the BIG-IP to delete it. Re-add the BIG-IP to F5 ACI ServiceCenter with the changed port. The BIG-IP data will still be retained after the delete and re-add.
+Click the delete (X) icon besides the BIG-IP to delete it. Re-add the BIG-IP to F5 ACI ServiceCenter with the changed port (For example, from the default 443 to 8443). The BIG-IP data will still be retained after the delete and re-add.
```

Comment:
either "beside" or "next to", not besides

Reply:
Fixed